src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/UsersController.cs

```diff
@@ -68,14 +68,14 @@ public async Task<ActionResult<CreateBuyerResponseViewModel>> CreateBuyerAsync(G
                 createBuyerRequest.EmailAddress
                 ));
 
-            if (!result.IsSuccess)
+            if (result.IsSuccess)
             {
-                response.Errors = result.Errors.Select(x => new ErrorMessageViewModel { Id = x.Id,  Field = x.Field });
-            
-                return BadRequest(response);
+                response.UserId = result.Value;
+                return Ok(response);
             }
             
-            return Ok(response);
+            response.Errors = result.Errors.Select(x => new ErrorMessageViewModel { Id = x.Id,  Field = x.Field });
```

Comment:
I think this is one for the coding standards. I have it on my list to add to the task. I treat them as a view model as I still class it as the "view" of the API. I have seen resource model used before. So far we have used the view model suffix for most of the endpoints so would like to stick with it for consistency (baring Ruth's latest PR).

I quite like the suffix as it makes searching for a class a lot easier. If we have two Organisation classes just separated by namespace one for the "view" & one for the "model" it can get a bit confusing & potentially leads to mistakes. So as long as it doesn't clash with the classes in the named in the Models folder I don't mind the 'Model' suffix.

Reply:
The problem here is that that view (sorrry!) is specific to you. In ASP.NET the term view model is normally associated with a particular aspect of the framework.

When I first joined the team I was baffled by all the references to view models when we didn't actually have any views...

I already had this one on my list for tomorrow!

Reply:
I'm not following so one to discuss 👍